Zend/zend_compile.c

```diff
@@ -4997,6 +5017,91 @@ static zend_result zend_try_compile_special_func(znode *result, zend_string *lcn
 	return zend_compile_frameless_icall(result, args, fbc, type) != (uint32_t)-1 ? SUCCESS : FAILURE;
 }
 
+static const char *zend_get_cstring_from_property_hook_kind(zend_property_hook_kind kind) {
+	switch (kind) {
+		case ZEND_PROPERTY_HOOK_GET:
+			return "get";
+		case ZEND_PROPERTY_HOOK_SET:
+			return "set";
+		EMPTY_SWITCH_DEFAULT_CASE()
+	}
+}
+
+static zend_string *zend_copy_unmangled_prop_name(zend_string *prop_name)
+{
+	if (ZSTR_VAL(prop_name)[0] != '\0') {
+		return zend_string_copy(prop_name);
+	} else {
+		const char *unmangled = zend_get_unmangled_property_name(prop_name);
+		return zend_string_init(unmangled, strlen(unmangled), /* persistent */ false);
+	}
+}
+
+static bool zend_compile_parent_property_hook_call(znode *result, zend_ast *ast, uint32_t type) /* {{{ */
```

Comment:
Don't know if anyone cares, but this comment (`/* {{{ */`) seems to be mismatched (no closing braces).

Reply:
Thanks. I fixed it.